src/components/Page/Page.jsx

```diff
@@ -77,6 +77,16 @@ export class Page extends Component {
     })
   }
 
+  _changeToolSetting(toolSettings,newValue) {
+    this.state.boardState.drawBoard.changeToolSetting(toolSettings,newValue);
+    this.setState({
+      boardState: {
+        ...this.state.boardState,
+        toolSettings
```

Comment:
What does this even do? :joy:

Reply:
it changes the previous state of the board by applying the new toolSettings right?😅

Reply:
But... Like... toolSettings is a string but that is a property and it needs a value and... What are you trying to do? :joy:

Reply:
what i am trying to do :  _changeToolSetting will gave a property name (i.e.,toolSettings) to be changed and we have a board property toolSettings so when both of key:value are same we can skip one of them right?
that's my intention

Reply:
what i am trying to do :  _changeToolSetting will gave a property name (i.e.,toolSettings) to be changed and we have a board property toolSettings so when both of key:value are same we can skip one of them right?
that's my intention

Reply:
Still don't understand :joy:. Does it work?

Reply:
No 🤣  just gave less error

Reply:
No 🤣  just gave less error

Reply:
look-

![Screenshot 2021-02-27 at 11 56 09 PM](https://user-images.githubusercontent.com/75980147/109396895-d6ef7680-7959-11eb-9989-f167674e25d0.jpg)

btw i will try to fix it tomorrow 😊 .

Reply:
look-

![Screenshot 2021-02-27 at 11 56 09 PM](https://user-images.githubusercontent.com/75980147/109396895-d6ef7680-7959-11eb-9989-f167674e25d0.jpg)

btw i will try to fix it tomorrow 😊 .

Reply:
ok gn :)

Reply:
i don't know why the `this.brushSizeRangeRef` is giving a null value 😫
 see -
![Screenshot 2021-02-28 at 2 45 42 PM](https://user-images.githubusercontent.com/75980147/109413490-b06e2180-79d3-11eb-8c93-fbed3b8fbc50.jpg)

Reply:
Maybe it has not been rendered.